public/index.html

```diff
@@ -87,7 +87,10 @@
 	</form>
 	<h2>Food Truck list:</h2>
 	<ul class="trucks-list"></ul>
+    <br>
+    <h4><a id="foodChoices" href="javascript:void(0)">Food type options:</a></h4>
```

Comment:
Using javascript:void(0) is outdated (any inline handlers are outdated) - instead don't use an anchor element if it's not actually a link, just mimic the behavior. For example, you can change the cursor to a pointer in css: cursor: pointer;